src/lighthouse.php

```diff
@@ -127,6 +127,19 @@
 
     'paginate_max_count' => null,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Pagination Default Count
+    |--------------------------------------------------------------------------
+    |
+    | Set the default count value for the @paginate directive.
+    | A setting of "null" mean that "defaultCount" argument needs to be set
+    | for each paginate directive, or to pass the "first" argument in the request.
+    |
+    */
+
+    'pagination_default_count' => null,
```

Comment:
How about we namespace the configuration options for pagination? This would be `pagination.default_count` then.

Reply:
Sound good!